orchestration-shared/src/main/java/uk/gov/di/orchestration/shared/entity/ClientRegistry.java

```diff
@@ -75,9 +78,23 @@ public ClientRegistry withClientName(String clientName) {
         return this;
     }
 
+    @DynamoDbAttribute("PublicKeySource")
+    public String getPublicKeySource() {
+        return Optional.ofNullable(publicKeySource).orElse(PublicKeySource.STATIC.getValue());
```

Comment:
Doesn't matter much here because it's a simple method with no side effects, but calling a method in an `orElse` is typically not what we want to do because it's evaluated even if the value is not needed. Instead we should do something like 
```suggestion
        return Optional.ofNullable(publicKeySource).orElseGet(PublicKeySource.STATIC::getValue);
```
which will only execute if it's needed

Reply:
Done